cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -108,6 +112,20 @@ import           Cardano.Api.SerialiseCBOR
 import           Cardano.Api.SerialiseTextEnvelope
 import           Cardano.Api.TxBody
 
+import qualified Cardano.Ledger.ShelleyMA.TxBody as Allegra
+
+-- | Get all the @Certificate@ from @Tx@.
+getTxCertificates :: forall era. Tx era -> Seq.StrictSeq Certificate
+getTxCertificates tx =
+    case getTxBody tx of
+        -- Byron doesn't have any additional certificates!
+        ByronTxBody {} -> Seq.Empty
+        ShelleyTxBody ShelleyBasedEraShelley (Shelley.TxBody _ _ certs _ _ _ _ _) _aux ->
```

Comment:
Why not use the `TxBody` record accessor `_certs`?

Reply:
A valid suggestion, Jordan.